wideq/client.py

```diff
@@ -138,6 +139,24 @@ def get_device(self, device_id) -> Optional['DeviceInfo']:
                 return device
         return None
 
+    def get_device_class(self, device_id):
+        """Look up a subclass of Device object by device ID.
+
+        Return a Device instance if no subclass exists for the device type.
+        Return None if the device does not exist.
+        """
+        from . import util
+
+        deviceInfo = self.get_device(device_id)
```

Comment:
`deviceInfo` -> `device_info`, for Pythonic style.

Reply:
ok, I come from java world, not the same style